### PR TITLE
Fix parsing new metadata format in resdk.tables.BaseTables.meta

### PR DESCRIPTION
**Pull request by JureZmrzlikar**

### CODE REVIEW

**Comment by JureZmrzlikar** [comment]
Fixes #284

**Review by AGregorc — APPROVED** [review]
lgtm

**Review by gregorjerse — APPROVED** [review]

File: src/resdk/tables/base.py [hunk 1]
```diff
@@ -402,17 +402,31 @@ def _get_orange_data(self) -> pd.DataFrame:
                 # TODO: logging, warning?
                 return pd.DataFrame()
 
-        if "mS#Sample ID" in df.columns:
+        if "Sample ID" in df.columns:
```
Comment (mstajdohar) [inline src/resdk/tables/base.py:405]: How about making this even more general? For example,
```
for key in df.columns:
    if "sample id" in key.lower()
        ...
```

Comment (JureZmrzlikar) [reply src/resdk/tables/base.py:405]: This PR is merged already. Valid point, i'll add this to one of the future PR's.

**Pull request merged**